core/controllers/acl_decorators_test.py

```diff
@@ -333,6 +355,8 @@ class CreateExplorationDecoratorTests(test_utils.GenericTestBase):
 
     class MockHandler(base.BaseHandler):
         GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+        URL_PATH_ARGS_SCHEMAS = {}
+        HANDLER_ARGS_SCHEMAS = {'GET': {}}
```

Comment:
```suggestion
    class MockBaseHandler(base.BaseHandler):
        URL_PATH_ARGS_SCHEMAS = {}
        HANDLER_ARGS_SCHEMAS = {'GET': {}}
   
    class MockHandler(MockBaseHandler):
        GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
        
```
For the MockHandlers with repeated schema, you can use a common base handler and inherit it. This will significantly reduce the repetition of code.

Reply:
@rohitkatlaa since our SVS functionality is designed in such a way that handler classes contains its schema locally, based on this I have written lint checks which verify that handlers have schema locally (i.e., URL_PATH_ARGS_SCHEMAS and HANDLER_ARGS_SCHEMA must be in node.locals ).
If I move common schemas of MockHandlers into a base class then it is raising lint errros.
Please correct me if I am missing anything.

Reply:
I think it is better to disable this particular lint check for the handlers in the test files. Essentially the lint check is to ensure the presence of schema in the handlers that are used in the API calls.

Reply:
I have a point to discuss where checking None value for  URL_PATH_ARGS_SCHEMAS and HANDLER_ARGS_SCHEMAS won’t work.
Example:
EditorAutoSaveHandler inherits the ExplorationHandler class. Since both of them are the child class of BaseHandler, now imagine a case when ExplorationHandler has a schema but EditorAutoSaveHandler does not. In this case if we can’t check whether HANDLER_ARGS_SCHEMA for EditorAutoSaveHandler is None since it inherits the schema for ExplorationHandler.
Please guide me if I am missing anything.

Reply:
I think it is fine to keep the schema for the test files. Thanks.